CONTRIBUTING.md

```diff
@@ -1,55 +1,160 @@
-# Contributing to knife-ec2
+# Contributing to Knife-EC2
 
-We are glad you want to contribute to Chef's knife-ec2 plugin! The first step is the desire to improve the project.
+We're glad you want to contribute to Knife-EC2! The first
+step is the desire to improve the project.
 
-You can find the answers to additional frequently asked questions [on the wiki](http://wiki.opscode.com/display/chef/How+to+Contribute).
+## Quick Contributing Steps
 
-## Quick-contribute
+1. Create an account on [GitHub](https://github.com).
+2. Create an account on the [Chef Supermarket](https://supermarket.chef.io/).
+3. [Become a contributor](https://supermarket.chef.io/become-a-contributor) by
+signing our Contributor License Agreement (CLA).
+4. Create a pull request for your change on [GitHub](https://github.com).
```

Comment:
This should point to the project or the pulls link, i.e. https://github.com/chef/knife-ec2/pulls